website/docs/advanced-concepts/advanced-integrations.md

```diff
@@ -0,0 +1,133 @@
+---
+sidebar_position: 0
+---
+
+# Integration - Advanced
+
+This section goes into detail on how [Integrations](../concepts/integration/) work. If you are just looking to use Functionless without extending it, this section can be skipped. If you have not read [Integrations](../concepts/integration/), do that first.
+
+## Terms
+
+- Resource - Infrastructure and Business Logic which use an integration, generally modelled as some infrastructure like a [`Function`](../concepts/function/) or [`EventBus`](../concepts/event-bridge/).
+- [Integration](../concepts/integration/) - An object (may be a Resource) or function which can be used within a Resource.
+- [Integration Type](./integration-types.md) - A pattern that an integration implements to be used within one or more Resources.
+
+```mermaid
+graph LR;
+  IntegrationType(Integration Type)
+  Resource--uses-->Integration;
+  Resource--supports-->IntegrationType;
+  Integration--implements-->IntegrationType;
+```
+
+A view of what a user sees of Integrations and Resources:
```

Comment:
Not sure what you mean by this?

Reply:
Trying to describe what these terms look like to someone using functionless.

starting from abstract terms and concepts and relating it to what it looks like when using the library.

Another attempt at phrasing:

> How Integrations and Resources appear as a consumer of Functionless